lib/core/test/integration/Test/Integration/Scenario/CLI/Wallets.hs

```diff
@@ -142,7 +164,52 @@ spec = do
         err `shouldBe` "Ok.\n"
         out `shouldNotContain` "CLI Wallet"
         c `shouldBe` ExitSuccess
-
   where
     emptyWallet' :: Context t -> IO String
     emptyWallet' = fmap (T.unpack . view walletId) . emptyWallet
+
+specPorts :: SpecWith (Context t)
+specPorts = do
+    it "PORTS_01 - Can create wallet with specifying port" $ \_ -> do
+        let walletName = "Wallet created via CLI"
+        checkSimpleCreateWallet
+            (Just "9080")
```

Comment:
Perhaps this can be:
```hs
        checkSimpleCreateWallet
            (Just $ show defaultPort)
```
If that is what you intended?

Reply:
Yes